src/main.c

```diff
@@ -285,11 +327,11 @@ int main(int argc, char *argv[]) {
 	if (r < 0)
 		exit_error(EXIT_FAILURE, r, "Failed loading config from '%s'", CONF_FILE);
 
-	n_pids = collect_pids(&pids);
+	n_pids = collect_pids(&pids, force);
 	if (n_pids < 0)
 		exit_error(EXIT_FAILURE, n_pids, "Failed collecting PIDs");
 	else if (n_pids == 0)
-		exit_error(EXIT_SUCCESS, 0, "Empty capture");
+		return EXIT_SUCCESS;
```

Comment:
nit: I'd keep the `"Empty capture"` message here for easier troubleshooting (despite it's implied by other messages).

Reply:
Changed in the new commit.